src/webview/js/js.js

```diff
@@ -908,6 +913,20 @@ documentBody.addEventListener('click', (e: MouseEvent) => {
   }
 });
 
+const revealMutedMessages = (target: Element) => {
```

Comment:
Separately, this should be somewhere up above the `click` listener. It's helpful that the `click` listener and this long-press handler are together pretty compact, with just a few lines for each of the different actions the user might take -- that's helpful in scanning through to see all the logic for dispatching between them.

Reply:
Done. I couldn't find a great place to put this, but I moved it up a bit.